NICE.Identity.Management/ClientApp/src/views/EditOrganisationUsers/EditOrganisationUsers.tsx

```diff
@@ -5,14 +5,14 @@ import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
 import { Grid, GridItem } from "@nice-digital/nds-grid";
 import { PageHeader } from "@nice-digital/nds-page-header";
 import { Table } from "@nice-digital/nds-table";
-import { Endpoints } from "../../data/endpoints";
-import { fetchData } from "../../helpers/fetchData";
-import { isDataError } from "../../helpers/isDataError";
-import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage";
-import { UsersAndJobIdsByOrganisationType, UserType } from "../../models/types";
-import { UserStatus } from "../../components/UserStatus/UserStatus";
-import { FilterSuggestions } from "../../components/FilterSuggestions/FilterSuggestions";
-import { SuggestionUser } from "../../components/SuggestionUser/SuggestionUser";
+import { Endpoints } from "src/data/endpoints";
+import { fetchData } from "src/helpers/fetchData";
+import { isDataError } from "src/helpers/isDataError";
+import { ErrorMessage } from "src/components/ErrorMessage/ErrorMessage";
+import type { UsersAndJobIdsByOrganisationType, UserType } from "src/models/types";
```

Comment:
not sure about best practice here, {type UsersAndJobIds... , type UserType} perhaps?